### ISSUE TITLE
Refactor coding style across project

### ISSUE THREAD

**Issue by Lercerss** [Lercerss/SOEN341W18#84]
## Description
Coding style should be properly enforced and refactored across all project files, regardless of markup or programming language.
  - Story points: 3
  - Priority: High
  - Risk: High

## Tasks
  1. Refactor coding style across all project python files according to [PEP-8](https://www.python.org/dev/peps/pep-0008/).
  1.  Refactor rendered templates and static files according to the [W3 validator](https://validator.w3.org/).

## Acceptance Test
  1. Open a python file.
  1. Notice that the code is formatted and styled according to PEP-8.
  1. Load a page on the website.
  1. Notice that the HTML is properly rendered according to W3 standards.

**Comment by Lercerss** [comment]
As referenced in the Sprint 3 evaluation: 
```
No reference for coding style
Inconsistent file naming convention: (e.g., answerspage.html, EditUserProfile.html, base_navigator.html)
"".good_answer"" in base.css, should it be "".good-answer""?
```

**Comment by Lercerss** [comment]
Implemented as part of #102.
